modules/nf-core/metabat2/metabat2/main.nf

```diff
@@ -45,4 +45,23 @@ process METABAT2_METABAT2 {
         metabat2: \$( metabat2 --help 2>&1 | head -n 2 | tail -n 1| sed 's/.*\\:\\([0-9]*\\.[0-9]*\\).*/\\1/' )
     END_VERSIONS
     """
+
+    stub:
+    def args             = task.ext.args   ?: ''
+    def prefix           = task.ext.prefix ?: "${meta.id}"
+    def decompress_depth = depth           ? "gzip -d -f $depth"    : ""
+    def depth_file       = depth           ? "-a ${depth.baseName}" : ""
+
+    """
+    echo "" | gzip -c > test.1.fa.gz
```

Comment:
Question rather than a comment - is it better to stream an empty file to gzip if you need a gzipped stub file?

Reply:
Yes [^1]

[^1]: https://nf-co.re/docs/guidelines/components/modules#stub-gzip-files-must-use-echo-and-pipe oh how I love that I annoy people to document this stuff properly... WORTH IT!

Reply:
Although that mayd me notice a mistake

Reply:
Ah! That's really smart 😁